tools/check_release_notes.sh

```diff
@@ -134,11 +134,11 @@ validate_opts() {
 # Curl the GitHub API to get a list of files for the specified PR. If files are
 # found, exit. We might eventually want to validate the data here.
 checkForFiles() {
-    echo "Checking files from pull request ${REPO_OWNER}/${REPO_NAME}#${PULL_NUMBER} head SHA: ${PULL_PULL_SHA} destination branch: ${PULL_BASE_REF}"
+    echo "Checking files from pull request ${REPO_OWNER}/${REPO_NAME}#${PULL_NUMBER} head SHA: ${PULL_PULL_SHA} destination branch: ${PULL_BASE_REF}, base SHA: ${PULL_BASE_REF}"
```

Comment:
```suggestion
    echo "Checking files from pull request ${REPO_OWNER}/${REPO_NAME}#${PULL_NUMBER} head SHA: ${PULL_PULL_SHA}, base SHA: ${PULL_BASE_REF}"
```

Reply:
I think this info might be useful somehow when looking at the logs?

Reply:
PULL_BASE_REF is a SHA,not a branch.